src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -523,7 +523,7 @@ base::Expression opBuilderHelperStringConcat(const std::any& definition)
         {
             std::string result {};
 
-            for (auto parameter : parameters)
+            for (const auto& parameter : parameters)
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

the loop variable's type is not a reference type; this creates a copy in each iteration; consider making this a reference